genesis/vis/batch_renderer.py

```diff
@@ -85,19 +85,19 @@ def build(self):
         if gs.backend != gs.cuda:
             gs.raise_exception("BatchRenderer requires CUDA backend.")
 
-        cameras = self._visualizer._cameras
+        self._cameras = gs.List([camera for camera in self._visualizer._cameras if not camera.debug])
```

Comment:
What about adding a `cameras` property to `visualizer`, that would filter out debug camera by default?